scripts/tests/test_bertlm_datasets.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import print_function
+
+import datetime
+import os
+import io
+import random
+
+from flaky import flaky
+import pytest
+
+from ..language_model.transformer_lm_data import WikiText2WordPiece, WikiText103WordPiece
+
+
+###############################################################################
+# Language model
+###############################################################################
+@pytest.mark.serial
```

Comment:
it shouldn't be necessary to run this test in serial

Reply:
Thanks for the suggestions. Addressed.